StatBot/Database/PostgresRepositories/PostgresUserRepository.cs

```diff
@@ -47,14 +47,13 @@ public User GetById(int id)
         }
 
         public void Save()
-        {
+        {            
             _context.SaveChanges();
         }
 
         public void Update(User user)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename parameter 'user' to 'o' to match the interface declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS927)